internal/services/network/application_gateway_resource.go

```diff
@@ -609,9 +753,11 @@ func resourceApplicationGateway() *pluginsdk.Resource {
 			},
 
 			"request_routing_rule": {
```

Comment:
Does this conflicts with the `routing_rule`?

Reply:
No, they can be configured together. But at least exists one.